azure-pipelines.yml

```diff
@@ -244,6 +246,10 @@ jobs:
         --test_tag_filters=-broken,-cw310,-verilator,-dv \
         --target_pattern_file="${TARGET_PATTERN_FILE}"
     displayName: Build & test SW
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathToPublish: '$(Build.ArtifactStagingDirectory)'
+      artifactName: bazel_trace.gz
```

Comment:
I would suggest using a more precise name to specify which bazel trace it is (sw_build).